### ISSUE TITLE
[IG] Amend registration pages (and associated scipt & database schema) to allow more informed and more thorough consent

### ISSUE THREAD

**Issue by davekilroy** [AppertaFoundation/Diadem#80]
lots of details to include here...

**Comment by ColinSloane** [comment]
Dave will look at adding new fields to the registration pages for becoming a diadem assessor
Add [organisation drop down CCG]
free text box in case no CCG
Add[practice name drop down]
+ button if they are connected to more than 1

**Comment by ColinSloane** [comment]
Dave will look at adding new fields to the registration pages for becoming a diadem recipient
Add [organisation drop down CCG]
free text box in case no CCG
Add[practice name drop down]

only able to be linked to one CCG

**Comment by ColinSloane** [comment]
Add a new field for ROLE of assessor

**Comment by ColinSloane** [comment]
Roland to chase up with Ewan the Consent documentation for Dave to add to DiADeM Auth

**Comment by ColinSloane** [comment]
Agreed to set aside the concept of linking to the Diadem recipient at registrations
